### PR TITLE
Ticket 40760: treatment and assay schemas - add log messages for troubleshooting

### PR DESCRIPTION
**Pull request by labkey-bpatel**
#### Rationale
Treatment and Assay schema links not showing on staging server, and unable to repro the issue locally, so adding log messages for the action in question for troubleshooting.

#### Changes
- Add log messages for investigation on ValidateStudySchemaLinkAction.
- Fix display for Groups and Methods which contains html tags.

### CODE REVIEW

**Review by XingY — COMMENTED** [review]
Please add TODO comment to remove those logging

File: src/org/labkey/cds/CDSController.java [hunk 1]
```diff
@@ -1041,18 +1041,22 @@ else if (isPost())
...
             WebdavService service = ServiceRegistry.get().getService(WebdavService.class);
-            WebdavResource resource = service.lookup(Path.parse(form.filename));
+            LOG.info("File path: " + Path.parse(form.getFilename()).toString()); //TODO: Added for investigating Ticket 40760, to be removed.
+            WebdavResource resource = service.lookup(Path.parse(form.getFilename()));
 
             if (resource != null)
             {
                 File requestedFile = resource.getFile();
+                LOG.info(requestedFile.getName() + " is present: " + (requestedFile != null));//TODO: Added for investigating Ticket 40760, to be removed.
+                LOG.info(requestedFile.getName() + " is readable: " + requestedFile.canRead());//TODO: Added for investigating Ticket 40760, to be removed.
```
Comment (XingY) [inline src/org/labkey/cds/CDSController.java]: if requestedFile is null, this would throw NPE.

**Review by XingY — CHANGES_REQUESTED** [review]
Looking at how this was implemented, it's suspicious to me that LABKEY.contextPath was prepended to href of those files on the client, but the filename (webdav path) param for the validation api does not have the LABKEY.contextPath. The prod/stage server has contextPath at root. Maybe before pushing a debug build to staging, you can try setting up your dev instance to have root context to see if it repros.

**Comment by labkey-bpatel** [comment]
No repro locally even with contextpath change to "root" from 'labkey'.

File: src/org/labkey/cds/CDSController.java (hunk 1 above)
Comment (labkey-bpatel) [reply src/org/labkey/cds/CDSController.java]: yes, good point, now updated.

**Review by XingY — APPROVED** [review]

**Pull request merged**